.github/workflows/check_docs_ci.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+BUILD_DIR=$1
+shift 1
+
+if [ -z "$BUILD_DIR" ]; then
+  echo "ERROR: BUILD_DIR should be argument 1."
+  exit 1
+fi
+
+CHECK_DOCS="$(cd "$(dirname "$0")"/../../scripts || exit 1; pwd)"/check_docs.sh
```

Comment:
We don't add `|| exit 1` elsewhere and the script will fail anyway if `CHECK_DOCS` doesn't point to `check_docs.sh`. Perhaps drop?

I also have a slight preference for just having `REPO_ROOT="$(cd "$(dirname "$0")/../.."); pwd)"` at the top of all these scripts so everything can be referred to consistently.

(One can get pretty sophisticated/cryptic with getting the path [1, 2]. Anything simple and clear is fine with me. Anything that needs to be robust probably shouldn't be done in bash anyway.)


[1] https://stackoverflow.com/questions/59895/how-to-get-the-source-directory-of-a-bash-script-from-within-the-script-itself
[2] https://stackoverflow.com/questions/4774054/reliable-way-for-a-bash-script-to-get-the-full-path-to-itself

Reply:
I'll take your simple consistent approach. I really wish the BSD runtime had `realpath`. Lacking it really makes scripting correctly harder.